ONSdigital/design-system
# Header logo tweaks

### What is the context of this PR?
Fixes: #1266 

### How to review 
- Check that when the window is at the xs breakpont (300px) the census header logo is the same size with or without the menu button
- Check that the stacked ONS header logo is the same size for both Welsh and English
